managed/util.go

```diff
@@ -22,3 +22,24 @@ func isDiskSizeValid(clusterTier string, diskSize int64) bool {
 	}
 	return true
 }
+
+func isDiskIopsValid(cloudType string, clusterTier string, diskIops int64) (bool, string) {
+	err := ""
+	if cloudType != "AWS" {
+		err = "Custom Disk IOPS is only supported for AWS"
+		return false, err
+	}
+	if clusterTier != "PAID" && diskIops != 3000 {
```

Comment:
May be move the 3000 check to the check at the end?

Reply:
Added more code to help clarify the purpose of this check. It's specifically for free tier clusters, so has no relation to the other checks.